config/src/config/network_config.rs

```diff
@@ -151,7 +149,8 @@ impl NetworkConfig {
             if !network_role.is_validator() && self.enable_remote_authentication {
                 ensure!(
                     self.peer_id == peer_id,
-                    "For full-nodes that use remote authentication, the peer_id must be derived from the identity key.",
+                    "For full-nodes that use remote authentication,
+                    the peer_id must be derived from the identity key.",
```

Comment:
Minor: Just in case you didn't realize this will cause a 2 line string of the form:
```
 For full-nodes that use remote authentication,
                    the peer_id must be derived from the identity key.
```
(See: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=282e9c9418d2e4cd26b75bdb5d84ae46)
It might be a little ugly

Reply:
right, should be 

```rust
                    "For full-nodes that use remote authentication, \
                     the peer_id must be derived from the identity key.",
```